benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -15025,7 +15025,7 @@ exports[`Render Summary for multi-table-only-columns.diff.html 1`] = `
        >
         <div
          className="cell"
-         dangerouslySetInnerHTML={".029"}
+         dangerouslySetInnerHTML={".0290"}
```

Comment:
Does `sum` count as a calculated value and should be padded?

Reply:
Similarly to `avg`, it would be better to not add more digits than in the inputs.

Reply:
Urrgghhh go home javascript, you are drunk
```
> 0.018+0.011
0.028999999999999998

```